mms/go/cmd/gocqlxapp/main.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"goapp/internal/log"
+	"goapp/internal/scylla"
+
+	"github.com/gocql/gocql"
+	"github.com/scylladb/gocqlx"
+	"github.com/scylladb/gocqlx/qb"
+	"github.com/scylladb/gocqlx/table"
+	"go.uber.org/zap"
+)
+
+var (
+	deleteStmt  string
+	deleteNames []string
+
+	insertStmt  string
+	insertNames []string
+
+	selectStmt  string
+	selectNames []string
+
+	tbl *table.Table
+)
+
+type Record struct {
+	FirstName       string `db:"first_name"`
+	LastName        string `db:"last_name"`
+	Address         string `db:"address"`
+	PictureLocation string `db:"picture_location"`
+}
+
+func init() {
```

Comment:
Init is a bad practice. Use struct instead

Reply:
Why is it bad for main apps?

Reply:
Removed init for specific global assignment.